tests/unauthorized/filesTest.js

```diff
@@ -1,34 +1,129 @@
+    const promises = []
+    promises.push(setGeneralInteractions(provider))
+    promises.push(provider.addInteraction({
+      uponReceiving: 'a request to get file contents with invalid authentication',
+      withRequest: {
+        method: 'GET',
+        path: webdavUrl,
+        headers: requestHeaderWithInvalidAuth
+      },
+      willRespondWith: unauthorizedResponse
+    }))
+    promises.push(provider.addInteraction({
+      uponReceiving: 'a request to put file contents with invalid authentication',
+      withRequest: {
+        method: 'PUT',
+        path: webdavUrl,
+        headers: requestHeaderWithInvalidAuth
+      },
+      willRespondWith: unauthorizedResponse
+    }))
+    promises.push(provider.addInteraction({
+      uponReceiving: 'a request to delete file contents with invalid authentication',
+      withRequest: {
+        method: 'DELETE',
+        path: webdavUrl,
+        headers: requestHeaderWithInvalidAuth
+      },
+      willRespondWith: unauthorizedResponse
+    }))
+    promises.push(provider.addInteraction({
+      uponReceiving: 'a request to create a directory with invalid authentication',
+      withRequest: {
+        method: 'MKCOL',
+        path: webdavUrl,
+        headers: {
+          authorization: invalidAuthHeader,
+          Origin: origin
+        }
+      },
+      willRespondWith: {
+        status: 401,
+        headers: webdavFilesResponseHeader,
+        body: incorrectAuthorizationXmlResponseBody
+      }
+    }))
+    promises.push(provider.addInteraction({
+      uponReceiving: 'a request to list files with invalid authentication',
+      withRequest: {
+        method: 'PROPFIND',
+        path: webdavUrl,
+        headers: requestHeaderWithInvalidAuth
+      },
+      willRespondWith: unauthorizedResponse
+    }))
+    promises.push(provider.addInteraction({
+      uponReceiving: 'a request to move file with invalid authentication',
+      withRequest: {
+        method: 'MOVE',
+        path: webdavUrl,
+        headers: requestHeaderWithInvalidAuth
+      },
+      willRespondWith: unauthorizedResponse
+    }))
+    promises.push(provider.addInteraction({
+      uponReceiving: 'a request to copy a file with invalid authentication',
+      withRequest: {
+        method: 'COPY',
+        path: webdavUrl,
+        headers: requestHeaderWithInvalidAuth
+      },
+      willRespondWith: unauthorizedResponse
+    }))
+
+    Promise.all(promises).then(done, done.fail)
   })
 
   afterAll(function (done) {
     provider.removeInteractions().then(done, done.fail)
   })
 
   // TESTING CONFIGS
-  var testContent = 'testContent'
-  var testFolder = '/testFolder' + timeRightNow
-  var testSubFiles = [
-    testFolder + '/' + 'file one.txt',
-    testFolder + '/' + 'zz+z.txt',
-    testFolder + '/' + '中文.txt',
-    testFolder + '/' + 'abc.txt',
-    testFolder + '/' + 'subdir/in dir.txt'
+  const testContent = config.testContent
```

Comment:
why redefining the variables and not just use `config.` in all places?

Reply:
done